### PR TITLE
Todos - Main

### PR DESCRIPTION
**Pull request by matej494**

### CODE REVIEW

File: Marsiple/App/Todos/TodosViewController.swift [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+//
+//  Created by Matej Korman on 26/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import SnapKit
+
+class TodosViewController: UIViewController {
+    // TODO: Replace todos with real data
+    private lazy var todos = setupTodos()
```
Comment (dkulundzic) [inline Marsiple/App/Todos/TodosViewController.swift:13]: Hmm, shouldn't this be fetched from the server?

**Review by dkulundzic — COMMENTED** [review]
Hmm, is this the `final` codebase for `Todos`? Or are we expecting new code and functionality?

File: Marsiple/App/Todos/TodosViewController.swift (hunk 1 above)
Comment (matej494) [reply Marsiple/App/Todos/TodosViewController.swift:13]: Yes it should. I saw that we have just main branch, and assumed it's done, but I need to implement fetching data.

Comment (dkulundzic) [reply Marsiple/App/Todos/TodosViewController.swift:13]: Indeed. Lets close this PR and proceed with implementing the data fetch.

**Pull request closed without merging**